### ISSUE TITLE
Links do not have descriptive text 

### ISSUE THREAD

**Issue by jackyzha0** [getreflect/get-reflect#34]
Descriptive link text helps search engines understand your content

This link should be updated: `https://forms.gle/ctypb6FmDT9RQqjv6`
